src/roles/nuh-deploy/tasks/main.yml

```diff
@@ -15,6 +15,24 @@
     ssh_host: "{{ hostname }}"
     host_username: "root"
 
+- name: Check if ha proxy is running
```

Comment:
Please add "(ignoring errors)" to name to allow to be filtered out of error checking

Reply:
done